src/controller/cachacaController.ts

```diff
@@ -0,0 +1,32 @@
+import { Request, Response } from 'express'
+import { z } from 'zod'
+
+import cachacaService from '../service/cachacaService'
+
+const CachacaController = {
+  async index(req: Request, res: Response) {
+    try {
+      const body: ChachacaDTO = AnoSchema.parse(req.body)
+
+      const response = await cachacaService.addMoradorAoRank(body.ano)
+
+      return res.send({
+        message: 'Sucesso',
+        rank: response
+      })
+
+    } catch (error) {
+
+      console.error(`Error[SERVER](${new Date().toDateString()}): Server error!`, error)
+
+      return res.status(500).json({ message: 'Something went wrong with server', error })
+    }
+  }
+}
+
+const AnoSchema = z.object({
```

Comment:
Aqui é só uma bilbioteca pra validar a entrada do usuário. Assim a gente traz erros melhores quando for obrigatório algum tipo de dado. Aqui acho que vai precisar só tirar o .object e deixar o number